curriculum/challenges/english/25-front-end-development/quiz-javascript-dates/66edd3711bb9f7efa73aef91.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
+DAY-MONTH-YEAR
 
 ---
 
-Placeholder distractor 2
+DAY-MONTH-YEAR:HOUR
 
 ---
 
-Placeholder distractor 3
+DAY-MONTH-YEAR HOUR:MINUTES:SECONDS
 
 #### --answer--
 
-Placeholder answer
+YEAR-MONTH-DAY HOUR:MINUTES:SECONDS:MILLISECONDS
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the output of `console.log(new Date(2003, 6, 27).getMonth());`?
 
 #### --distractors--
 
-Placeholder distractor 1
+27/6/2003
 
 ---
 
-Placeholder distractor 2
+27
 
 ---
 
-Placeholder distractor 3
+2003
 
 #### --answer--
 
-Placeholder answer
+6
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How would you format a date to a locale-specific string or a more readable format?
 
 #### --distractors--
 
-Placeholder distractor 1
+`.toLocaleDate()`
 
 ---
 
-Placeholder distractor 2
+`.toLocaleString()`
 
 ---
 
-Placeholder distractor 3
+`.toLocaleTimetring()`
 
 #### --answer--
 
-Placeholder answer
+`.toLocaleDateString()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+The `.toLocaleDateString()` date format looks like?
 
 #### --distractors--
 
-Placeholder distractor 1
+MONTH / DAY / SECONDS
 
 ---
 
-Placeholder distractor 2
+MONTH / DAY
 
 ---
 
-Placeholder distractor 3
+YEAR / MONTH / DAY
 
 #### --answer--
 
-Placeholder answer
+MONTH / DAY / YEAR
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does `new Date().getDay()` return?
 
 #### --distractors--
 
-Placeholder distractor 1
+It returns the current day of the week as a string
 
 ---
 
-Placeholder distractor 2
+It returns the current day of the year as a boolean.
 
 ---
 
-Placeholder distractor 3
+It returns the current day of the month as a decimal.
 
 #### --answer--
 
-Placeholder answer
+It returns the current day of the week as a number.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the output of `console.log(new Date(2003, 6, 27).getFullYear());`?
 
 #### --distractors--
 
-Placeholder distractor 1
+27/6/2003
 
 ---
 
-Placeholder distractor 2
+27
 
 ---
 
-Placeholder distractor 3
+6
 
 #### --answer--
 
-Placeholder answer
+2003
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+The `toUTCString()` method formats a date as?
 
 #### --distractors--
 
-Placeholder distractor 1
+A door dash string format
 
 ---
 
-Placeholder distractor 2
+An MP3 string format
 
 ---
 
-Placeholder distractor 3
+An ISO-8601 string format
 
 #### --answer--
 
-Placeholder answer
+A UTC string format
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+If we are in the month of October this code `console.log(new Date().getMonth());` outputs what ?
 
 #### --distractors--
 
-Placeholder distractor 1
+1
 
 ---
 
-Placeholder distractor 2
+3
 
 ---
 
-Placeholder distractor 3
+10
 
 #### --answer--
 
-Placeholder answer
+9
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+If the ouput of this code is "2021", what date method is missing from `console.log(new Date("2021-12-25")(MISSING METHOD))`;
```

Comment:
````suggestion
Which option will output `"2021"` for the following object?

```js
const d = new Date("2021-12-25");
```
````
I think something like this will make it easier to understand the question.

Reply:
Yes it would